upstairs/src/deferred.rs

```diff
@@ -0,0 +1,186 @@
+// Copyright 2023 Oxide Computer Company
```

Comment:
2024!!!!!!!!!!

Reply:
Fixed in 85b6994675fbe104475376eee6f21e2616e1ef16, WELCOME TO THE FUTURE